### ISSUE TITLE
conda build 2.0.0beta2-py27_0 ImportError

### ISSUE THREAD

**Issue by dsludwig** [conda/conda-build#1244]
```
conda build conda.recipe
Traceback (most recent call last):
  File "/home/ubuntu/miniconda2/bin/conda-build", line 4, in <module>
    import conda_build.main_build
ImportError: No module named main_build

conda build conda.recipe returned exit code 1
```

It looks like the `meta.yaml` is referring to `conda_build.main_build` instead of `conda_build.cli.main_build` (this occurs in a few other places).

**Comment by msarahan** [comment]
Thanks, working on this now.

**Comment by msarahan** [comment]
Should be fixed (and tested) in 2.0.0beta3.  Please let me know if it is still misbehaving.  Closing for now.
